docs/installation.rst

```diff
@@ -32,31 +32,34 @@ From version 2.0.0, sub commands are changed.
-|                   |                     | New command only can       |
-|                   |                     | take --kde option          |
-+----------------- -+---------------------+----------------------------+
-| install-doc       | doc                 | version moved after target |
-+----------------- -+---------------------+----------------------------+
-|                   | list                | legacy list commands are   |
-|                   |                     | removed.                   |
-+----------------- -+---------------------+----------------------------+
-| list-qt           |                     |                            |
-+----------------- -+---------------------+----------------------------+
-| list-tool         |                     |                            |
-+----------------- -+---------------------+----------------------------+
++------------------+---------------------+----------------------------+
+| New sub commands | Legacy sub commands |  Note                      |
++==================+=====================+============================+
+| install-qt       | install             | Version moved after target |
++------------------+---------------------+----------------------------+
+| install-tool     | tool                | Arguments are changed      |
+|                  |                     |                            |
+|                  |                     | New syntax doesn't take    |
+|                  |                     | version                    |
++------------------+---------------------+----------------------------+
+| install-example  | examples            | Version moved after target |
+|                  |                     |                            |
+|                  |                     | Caution with last (s)      |
++------------------+---------------------+----------------------------+
+| install-src      | src                 | Version moved after target |
+|                  |                     |                            |
+|                  |                     | New command only can       |
+|                  |                     | take --kde option          |
++------------------+---------------------+----------------------------+
+| install-doc      | doc                 | Version moved after target |
++------------------+---------------------+----------------------------+
+|                  | list                | Legacy list commands are   |
+|                  |                     | removed.                   |
++------------------+---------------------+----------------------------+
+| list-qt          |                     |                            |
++------------------+---------------------+----------------------------+
+| list-tool        |                     |                            |
++------------------+---------------------+----------------------------+
 
 
 Usage
```

Comment:
Now  there is `Getting Started` section, so `Usage` is a duplicated so we can remove it.

Reply:
It seems better to move Output directory option section to a last of getting started section.

Reply:
This is a really good idea, thank you!